eslint-warning-thresholds.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Created to ensure we aren't introducing more eslint violations thanks to @mcmire's implementation in Core's eslint fix PR https://github.com/MetaMask/core/pull/5132